### ISSUE TITLE
Enhancement: Increase "short format" for Locations

### ISSUE THREAD

**Issue by xyeLz** [Pterodactyl/Panel#313]
A couple issues/requests here (Note: I'm currently using 0.6-pre.4).

Back in 0.5.7, the short format for the locations was limited to 10. Thus, I couldn't create a location longer than 10 characters. It looks like this has been limited to 20 in 0.6, but I occasionally will surpass this limit. I know where I can change this field, but it would be nice if it could be built into the config (so I don't need to remember to change this during each upgrade). Would it be possible to increase this value to something like 32, so long as it's not breaking anything else?

Additionally, in the current version of the panel, and this may be a moot point with the new GUI, but the error I get when trying to create a location longer than 20 characters isn't very descriptive:

```strings.whoops! auth.errorencountered
The short format is invalid.```

This would be very helpful for me personally and I don't see why it couldn't be implemented by default so long as it doesn't break anything.

Thank you!

**Comment by DaneEveritt** [comment]
The point of the short format is to be short. `us1` is a short format, and you can use the description field to actually describe the location. I don't see any reason to have a 32 character short location field.

Even `do-nyc-2` is a solid short location name.

**Comment by xyeLz** [comment]
We might end up agreeing to disagree on this one. Maybe 32 characters is a bit much, but even something like 24 or 26 would be closer to what I would prefer. I understand it's the "short" format for a reason, but I have a very specific method for my short formats. For example, I identify generally at the region level and slowly specify into the exactly location as to make sorting much simpler:

Long Version: Atlanta, Georgia; United States; North America
Short Version: na.us.east.atlanta

It doesn't seem like much, but that's already 18 characters. If I wanted to add Houston, I would already be over the limit:

na.us.central.houston

Again, perhaps I am in the minority here by not simply using a one-liner. I just figured if it was an easy change on your end and wouldn't affect the performance of the panel in any way, why would it hurt to make the change?

**Comment by DaneEveritt** [comment]
Thanks for the examples. I'll raise the limits.

**Comment by xyeLz** [comment]
That's awesome. Thank you!

**Comment by DaneEveritt** [comment]
Bumped to 60 character limit.

**Comment by gitdragon0200** [comment]
what kind of location i can try
